server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -36,11 +36,8 @@ def create
     end
 
     def update
-      # update_attribute does not call validation, which is desired, while
-      # update_attributes does. Horay consistency!
-      for name, value in params[:deployment]
-        @deployment.update_attribute(name, value)
-      end
+      @deployment.attributes = params[:deployment]
+      @deployment.save(:validate => false)
```

Comment:
@sdherr, I think this will work, but I haven't tested it other than in the rails console. This will generate only 1 SQL call whereas your loop will generate several SQL calls.